sdk-tests/src/test/java/io/dapr/it/actors/app/TestController.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.it.actors.app;
+
+import io.dapr.actors.runtime.ActorRuntime;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+/**
+ * SpringBoot Controller to handle callback APIs for Dapr.
+ */
+@RestController
+class TestController {
```

Comment:
For now, reuse the existing controller and add your actor type to the list. I have a small refactoring that will allow us to reuse DaprApplication.

Reply:
The issue is the existing DaprApplication overrides the config call with non-default settings and I want default settings.

Reply:
OK.